serverFiles/controllers/articles.controllers.js

```diff
@@ -0,0 +1,17 @@
+const selectArticlesById = require('../models/articles.models');
+
+function getArticlesById(req, res, next) {
+  const { article_id } = req.params;
+  const parsedArticleId = parseInt(article_id);
+  if (!parsedArticleId) {
+    res.status(400).send({ msg: 'Bad request' });
+  } else {
```

Comment:
When would this not be a parsed article? Your 400 error should be caught in your catch block.

Reply:
i believe my error is now caught in my controller catch block